hyperdome/client/add_server_dialog.py

```diff
@@ -123,14 +124,29 @@ def add_server(self):
         self.add_server_button.setText("Checking...")
 
         probe = threads.ProbeServerTask(self.session, self.server)
-        probe.signals.success.connect(self.set_server)
+        if self.server.is_counselor:
+            probe.signals.success.connect(self.signup)
+        else:
+            probe.signals.success.connect(self.set_server)
```

Comment:
Ternary is cool tho

Reply:
ooh yeah, I had multiple lines in those for a couple commits, didn't think to refactor